docs/components/checkbox-field/index.md

```diff
@@ -12,7 +12,18 @@ Provide a string to `@hint` to render the text into the Hint section of the Fiel
 
 ## Error
 
-Provide a string to `@error` to render the text into the Error section of the Field. This is optional.
+Provide a string or array of strings to `@error` to render the text into the Error section of the Field. This is optional.
+
+```hbs
+<Form::CheckboxField @label='Single error' @error='Error' />
+```
+
+```hbs
+<Form::CheckboxField
+  @label='Multiple errors'
+  @error={{(array 'Error 1' 'Error 2')}}
+/>
```

Comment:
I feel as though `@error` having the option to be both single and plural is not great, but I don't have a better name. I think we definitely want to allow either `string | array<string>` though, so 🤷

Reply:
Maybe this can be updated to `errors` to match ember-headless-form: https://github.com/CrowdStrike/ember-toucan-core/pull/70/files#diff-9daa8ab5bdba22a48199ecf32b85e898ad08bcff962dc6824c40cc068aca25d5R18

But I think that can be a follow up to make it easier to read the changes.

Reply:
Hm, either way it can be confusing (a little bit at least), like `@errors="Fill in this field"` is also not what you would probably expect. 

I think it's ok the way it is, at least I have no better suggestion...